### ISSUE TITLE
AttributeError: getMibSymbol

### ISSUE THREAD

**Issue by cpackham** [lextudio/pysnmp#134]
### Expected behavior

We had an internal tool using pysnmp to parse some info out of some enterprise mibs. This has been working fine with versions from 7.1.0 up to 7.1.6

### Actual behavior

With the recent 7.1.7 release we start seeing the following error
```
  ./tools/check-mibs:148: DeprecationWarning: addMibCompiler is deprecated. Please use add_mib_compiler instead.
    compiler.addMibCompiler(mibBuilder, destination=tmpdir,
  ./tools/check-mibs:150: DeprecationWarning: loadModules is deprecated. Please use load_modules instead.
    mibBuilder.loadModules('AT-BOARDS-MIB', 'AT-PRODUCT-MIB')
  ./tools/check-mibs:38: DeprecationWarning: resolveWithMib is deprecated. Please use resolve_with_mib instead.
    objectIdentity.resolveWithMib(mvc)
  Traceback (most recent call last):
    File "./tools/check-mibs", line 156, in <module>
      board_ok = check_board_name(mibViewController, p)
    File "./tools/check-mibs", line 39, in check_board_name
      name = board_name(objectIdentity)
    File "./tools/check-mibs", line 31, in board_name
      sym = objectIdentity.getMibSymbol()
    File ".venv/lib/python3.9/site-packages/pysnmp/smi/rfc1902.py", line 776, in __getattr__
      raise AttributeError(attr)
  AttributeError: getMibSymbol
```

Looks like we probably do need to update our script to avoid the deprecated functions but it does seem strange to start getting deprecations in 7.1.7 and not 7.2.0. 

### Detailed steps

unsure

### Python package information

7.1.7

### Operating system information

Linux (ubuntu 22.04)

### Python information

3.9.15

**Comment by lextudio-support** [comment]
New issues are marked as low priority by default. [Becoming our commercial customers](https://www.pysnmp.com/support#commercial-support), and then your reports are handled with higher priority after triage.

**Comment by lextm** [comment]
Fixed in release 7.1.8.

**Comment by cpackham** [comment]
Awesome thanks.
